config/LamyEraser.conf

```diff
@@ -30,6 +30,7 @@ double-click                undo
 triple-click
 quadruple-click
 quintuple-click
+long-click                  redo
```

Comment:
Perhaps it's better to leave this blank in the default config so users aren't surprised by the behaviour?